server/Knowledge_checklist.sql

```diff
@@ -0,0 +1,100 @@
+drop table if exists students cascade;
+drop table if exists mentors cascade;
+drop table if exists region;
+drop table if exists learning_objectives cascade;
+drop table if exists mapping_skills cascade;
+drop table if exists tech_skills cascade;
+drop table if exists competency_level cascade;
+
+CREATE TABLE students (
+    id          SERIAL PRIMARY KEY,
+    name        VARCHAR (50)NOT NULL,
+    location    VARCHAR (50)NOT NULL,
+    email       VARCHAR (150) NOT NULL,
+    password    VARCHAR(70) NOT NULL
+);
+
+CREATE TABLE mentors (
+    id          SERIAL PRIMARY KEY,
+    name        VARCHAR (50)NOT NULL,
+    location    VARCHAR (50)NOT NULL,
+    email       VARCHAR (150) NOT NULL,
+    password    VARCHAR(70) NOT NULL
+);
+
+CREATE TABLE region (
+    id          SERIAL PRIMARY KEY,
+    location    VARCHAR(50) NOT NULL 
+);
+
+CREATE TABLE learning_objectives (
+    id           SERIAL PRIMARY KEY,
+    obj_id       VARCHAR (50)NOT NULL,
+    lesson_id    INT NOT NULL            
```

Comment:
I guess `lesson_id` is a foreign key to another table, but I could not find a `lessons` table.

Reply:
AO amended